es6-shim.js

```diff
@@ -1438,7 +1438,7 @@
       if (number === 0) {
         return 32;
       }
-      return 32 - (number).toString(2).length;
+      return 32 - 1 - Math.floor(Math.log(number) * Math.LOG2E + (1 / 4294967296));
```

Comment:
What does `4294967296` represent? Could `1 / 4294967296` be stored in a named constant outside the shim function?

Also, why not `31 - `?

Some comments explaining why this works would be helpful also.